tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java

```diff
@@ -104,17 +100,27 @@ private ServicesSupervisor connectRemote() throws Exception
     {
         ServicesSupervisor supervisor = null;
         String[] result = null;
+        PortalServerBehavior serverBehavior = null;
 
         try
         {
+            /* disconnect bundleSuperVisor and redirect outputstream */
+            serverBehavior = (PortalServerBehavior) _server.loadAdapter( PortalServerBehavior.class, null );
```

Comment:
Lets try to reuse the existing bundleSupervisor if possible, we may need to make the bundleSupervior more flexible.